server/client.go

```diff
@@ -10,9 +10,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
```

Comment:
I have no idea what happened with this file.. I simply rebased locally. Will fix